package.json

```diff
@@ -32,6 +32,7 @@
     "ramda": "^0.23.0",
     "rc-slider": "^6.1.0",
     "react": "^15.4.0",
+    "react-addons-shallow-compare": "^15.6.0",
```

Comment:
if the line isn't there if somebody cloned the repo, they would run into an error when they try to `npm i`